kindlecomicconverter/comic2ebook.py

```diff
@@ -650,6 +651,81 @@ def imgFileProcessing(work):
         return str(sys.exc_info()[1]), sanitizeTrace(sys.exc_info()[2])
 
 
+def mupdf_pdf_render_page(args):
```

Comment:
Would it be faster to render a page range per process instead of a single page per process?

https://pymupdf.readthedocs.io/en/latest/recipes-multiprocessing.html

Reply:
Yes, it's MUCH faster, like 15 sec to 4 sec